src/wikilink/context.rs

```diff
@@ -0,0 +1,59 @@
+use crate::{utils::string::kebab, Note, HashMap};
+use pest::Parser;
+
+pub struct WikilinkContext<'a> {
```

Comment:
```suggestion
#[derive(Debug)
pub struct WikilinkContext<'a> {
```

This helps with debugging custom handlers.